server.go

```diff
@@ -163,8 +163,23 @@ func newServer(listenAddrs []string, notifier chainntnfs.ChainNotifier,
 		return nil, fmt.Errorf("default listener must be TCP")
 	}
 
+	// TODO(roasbeef): remove once we actually sign the funding_locked
+	// stuffs
+	fakeSigHex, err := hex.DecodeString("30450221008ce2bc69281ce27da07e66" +
+		"83571319d18e949ddfa2965fb6caa1bf0314f882d70220299105481d63e0f" +
+		"4bc2a88121167221b6700d72a0ead154c03be696a292d24ae")
+	if err != nil {
+		return nil, err
+	}
+
+	fakeSig, err := btcec.ParseSignature(fakeSigHex, btcec.S256())
+	if err != nil {
+		return nil, err
+	}
+
 	chanGraph := chanDB.ChannelGraph()
 	self := &channeldb.LightningNode{
+		AuthSig:    fakeSig,
```

Comment:
This may be addressed the next commit, but if it isn't, then we should be generating a valid signature here for our node announcement. 

If it's done in the next commit, then you can safely ignore me here